habitat_baselines/config/test/ppo_imagenav_test.yaml

```diff
@@ -13,6 +13,9 @@ LOG_INTERVAL: 100
 CHECKPOINT_INTERVAL: 1
 
 RL:
+  POLICY:
+    name: "PointNavBaselinePolicy"
```

Comment:
Please, add default "POLICY.name" value to the default config [here](https://github.com/facebookresearch/habitat-api/blob/master/habitat_baselines/config/default.py#L51). Then we can drop adding policy in part of configs. I would set default to `PointNavBaselinePolicy` as PPO trainer is default one.

Reply:
Done!